x/ccv/provider/unbonding_test.go

```diff
@@ -435,71 +290,68 @@ func undelegate(s *ProviderTestSuite, delAddr sdk.AccAddress, valAddr sdk.ValAdd
 	return valsetUpdateID
 }
 
-// incrementTimeByProviderUnbondingPeriod increments the overall time by
-// the unbonding period on the provider; note that it is expected for this
-// period to be larger than the unbonding period on the consumer
-func incrementTimeByProviderUnbondingPeriod(s *ProviderTestSuite) {
+// incrementTimeByUnbondingPeriod increments the overall time by
+// 	- if provider == true, the unbonding period on the provider;
+//	- otherwise, the unbonding period on the consumer.
+// Note that it is expected for the provider unbonding period
+// to be one day larger than the consumer unbonding period.
+func incrementTimeByUnbondingPeriod(s *ProviderTestSuite, provider bool) {
```

Comment:
I think it's better to use an enum or another way to differentiate cases, instead of having `provider bool`. I think we should have an enum or just two constant strings in the tests `const P = "provider"; const C = "consumer"; and differentiate with a function param `chain string`

```
if chain == P{
...
}
```